Sources/TokamakDOM/DOMRenderer.swift

```diff
@@ -163,6 +163,6 @@ final class DOMRenderer: Renderer {
     guard mapAnyView(host.view, transform: { (html: AnyHTML) in html }) != nil
     else { return }
 
-    _ = parent.ref.removeChild!(target.ref)
+    try? _ = parent.ref.throwing.removeChild!(target.ref)
```

Comment:
I'm surprised this actually compiles, what are your thoughts on this version?

```suggestion
    _ = try? parent.ref.throwing.removeChild!(target.ref)
```

Reply:
As I understand it, there's no difference in behavior between the two. But if you prefer your version for stylistic reasons, I have no objections.